packages/functional-tests/pages/login.ts

```diff
@@ -320,9 +323,9 @@ export class LoginPage extends BaseLayout {
   }
 
   async submit() {
-    const waitForNavigation = this.page.waitForNavigation();
+    // const waitForNavigation = this.page.waitForNavigation();
     await this.page.locator(selectors.SUBMIT).click();
-    return waitForNavigation;
+    // return waitForNavigation;
```

Comment:
Wanted to see if removing this made anything fail, for some reason when this is left my new tests don't pass. It just hangs after clicking submit on the account confirmation page.

Reply:
Did we make the change here so that all the failing tests works now? @LZoog

Reply:
Yes that's right! [See this comment](https://github.com/mozilla/fxa/pull/15379/files#diff-037c452e31563d309e44bd84e10777b11bd359d75b436cbbf029adad8ad5b292R345-R347).

Whenever there's more than a couple of parameters passed like we were doing, an `options` object can start to feel a bit nicer so I did go ahead and have to make those changes across files.